### ISSUE TITLE
SDDM 0.16.0-1 crashes with systemd

### ISSUE THREAD

**Issue by tSte** [sddm/sddm#915]
After last update, I encounter issue while booting - SDDM won't start. It occurs randomly (~40%). My journal contains this:
```
sddm-helper[13788]: pam_systemd(sddm-greeter:session): Failed to create session: Refusing activation, D-Bus is shutting down.
sddm-greeter[13790]: Could not connect to any X display.
```

When I switch to tty2 and restart SDDM `systemctl restart sddm`, it works. It might be related to #905 but I am not sure.

I am on Arch 64 with 4.9.56-1-lts kernel.
SDDM 0.16.0-1
systemd 235.0-1
pam 1.3.0-1

**Comment by sanosis** [comment]
Try adding a delay to sddm - if it occurs randomly then 2-3s might be enouth
[https://github.com/sddm/sddm/issues/905#issuecomment-345622360](url)

**Comment by tSte** [comment]
#905 seems to be solved in 0.16.0-3....
